desc/integrals/singularities.py

```diff
@@ -316,6 +316,8 @@ class _BIESTInterpolator(IOAble, ABC):
         "_shift_z",
     ]
 
+    _static_attrs = ["_q"]
```

Comment:
why not also have st sz here?

Reply:
technically `st`/`sz` don't need to be static since they're not used for array sizes like `q`. In theory they don't even need to be integers